lib/sqs/index.js

```diff
@@ -121,6 +127,11 @@ module.exports = ({
   }
 
   return {
+    /**
+     * lists queues
+     */
+    listQueues: () => SQS.listQueuesAsync(),
```

Comment:
only test coverage missed in the whole library.  Might just want to cover it for completion.

Reply:
done.  I swore I ran test coverage and was surprised i saw 100%.....